modules/mediaConsortiumBidAdapter.js

```diff
@@ -102,3 +158,35 @@ function isValidResponse(response) {
       isPlainObject(response.body) &&
       isArray(response.body.bids)
 }
+
+function sendEventToGA4(eventName, additionalPayload, customTimestamp) {
+  if (!additionalPayload) {
+    additionalPayload = {};
+  }
+
+  gtag('get', 'G-HZ5RJ58ZF9', 'client_id', (clientId) => {
+    const payload = {
+      client_id: clientId,
+      events: [{
+        name: eventName,
+        params: {
+          ...additionalPayload,
+          session_id: localStorage.getItem('prebid.session_id'),
+          time_stamp: customTimestamp ? new Date(customTimestamp).toISOString() : new Date().toISOString(),
```

Comment:
timestamp should be initialized at the start of the function, because `gtag('get', ` takes some time.

(Though I also had the same mistake at https://github.com/mediaconsortium/Prebid.js/commit/315f898cfc2c0820d6825682c0f2d3507b348f5f#diff-3274f1a37032fb0ae4e2823def0007c634e869ae0dfc304ff6a12c36513c3a52R1099-R1100)